rdr_service/model/bq_pdr_participant_summary.py

```diff
@@ -325,7 +325,8 @@ class BQPDRModuleView(BQView):
            nt.mod_module, nt.mod_baseline_module,
            CAST(nt.mod_authored AS DATETIME) as mod_authored, CAST(nt.mod_created AS DATETIME) as mod_created,
            nt.mod_language, nt.mod_status, nt.mod_status_id, nt.mod_response_status, nt.mod_response_status_id,
-           nt.mod_external_id
+           nt.mod_external_id, nt.mod_questionnaire_response_id, nt.mod_consent, nt.mod_consent_value,
```

Comment:
Thanks for catching this.  Are we sure this is the only managed view where the SELECT statement is explicitly populated vs. "auto-generated" field list in the SELECT?  Since we're nuking all tables / views including genomics, etc.?

Reply:
I don't expect this to hit in prod, I think it's just some low quality questionnaire response data in the lower environments causing this to hit.  I don't think there will be a lot of hits in the lower environments.